react/src/pages/VFolderNodeListPage.tsx

```diff
@@ -0,0 +1,527 @@
+                  })}
+                  <Tooltip title={t('data.folders.Delete')} placement="topLeft">
+                    <Button
+                      style={{
+                        color: token.colorError,
+                      }}
+                      icon={<TrashBinIcon />}
+                      onClick={() => {}}
+                    />
+                  </Tooltip>
+                </>
+              )}
+            </Flex>
+          </Flex>
+          <VFolderNodes
+            loading={deferredQueryVariables !== queryVariables}
+            vfoldersFrgmt={filterNonNullItems(
+              _.map(vfolder_nodes?.edges, 'node'),
+            )}
+            rowSelection={{
+              type: 'checkbox',
+              // Preserve selected rows between pages, but clear when filter changes
+              preserveSelectedRowKeys: true,
+              getCheckboxProps(record: VFolderNodeInList) {
+                return {
+                  disabled: isDeletedCategory(record.status),
+                };
+              },
+              onChange: (selectedRowKeys) => {
+                // Using selectedRowKeys to retrieve selected rows since selectedRows lack nested fragment types
+                handleRowSelectionChange(
+                  selectedRowKeys,
+                  filterNonNullItems(_.map(vfolder_nodes?.edges, 'node')),
+                  setSelectedFolderList,
+                );
+              },
+              selectedRowKeys: _.map(selectedFolderList, (i) => i.id),
+            }}
+            pagination={{
+              pageSize: tablePaginationOption.pageSize,
+              current: tablePaginationOption.current,
+              total: vfolder_nodes?.count ?? 0,
+              showSizeChanger: true,
+              pageSizeOptions: ['10', '20', '50'],
+              showTotal: (total) => (
+                <Typography.Text type="secondary">
+                  {t('general.TotalItems', { total: total })}
+                </Typography.Text>
+              ),
+            }}
+            onChange={({ current, pageSize }, filters, sorter) => {
+              if (_.isNumber(current) && _.isNumber(pageSize)) {
+                setTablePaginationOption({ current, pageSize });
+              }
+              setQuery(
+                { order: transformSorterToOrderString(sorter) },
+                'replaceIn',
+              );
+            }}
+          />
+        </Flex>
+      </BAICard>
+      <InviteFolderPermissionSettingModal
+        onRequestClose={() => {
+          setInviteFolderId(null);
+        }}
+        vfolderId={inviteFolderId}
+        open={inviteFolderId !== null}
+      />
+      <ImportFromHuggingFaceModal
+        open={isVisibleImportFromHuggingFaceModal}
+        onRequestClose={() => {
+          toggleImportFromHuggingFaceModal();
+        }}
+      />
+      <FolderCreateModal
+        open={isOpenCreateModal}
+        onRequestClose={() => openCreateModal()}
```

Comment:
Please rename `openCreateModal()` to make it clear that it is a function for closing, as the current name is confusing.

Also, after a successful creation, update the counts and the current list.